man/systemd.exec.xml

```diff
@@ -225,6 +225,53 @@
         <xi:include href="system-only.xml" xpointer="singular"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProcMode=</varname></term>
+
+        <listitem><para>Takes one of <literal>noaccess</literal>, <literal>invisible</literal>,
+        <literal>ptraceable</literal> or <literal>default</literal>. Defaults to the latter. This controls
```

Comment:
"latter" is normally used with two things: "Of two things, the one mentioned second." [Webster]